test/Makefile

```diff
@@ -4,8 +4,8 @@ BINS=siolpain fmt stack cas hook hook2 stress1 fmtarr
 BINS+=bench-expire expire dumptwice syncget nsabuse drain merge
 BINS+=chkkey
 
-CFLAGS+=-I../../src -I ../src ${USER_CFLAGS}
-LDFLAGS+=../src/libsdb.a ${USER_LDFLAGS}
+CFLAGS+=$(shell pkg-config --cflags sdb) ${USER_CFLAGS}
```

Comment:
this builds against system wide sdb, not the one in current builddir, i understand that in meson the buildding is not the srcdir, but with this change, we are forced to use pkg-config and install it in the system to work.

i would prefer to be able to `make; make test; make install` rather than swaping the last 2 actions